tools/istio-iptables/pkg/cmd/run.go

```diff
@@ -436,7 +428,9 @@ func (iptConfigurator *IptablesConfigurator) run() {
 	}
 
 	iptConfigurator.handleInboundIpv4Rules(ipv4RangesInclude)
-	iptConfigurator.handleInboundIpv6Rules(ipv6RangesExclude, ipv6RangesInclude)
+	if iptConfigurator.cfg.EnableInboundIPv6s != nil {
```

Comment:
I think we should this same check on L316 where `ip6tables-save` is invoked.

Reply:
https://github.com/istio/istio/blob/master/tools/istio-iptables/pkg/cmd/run.go#L316

Reply:
Done.